ui/analyse/src/ceval/sunsetterProtocol.js

```diff
@@ -7,77 +7,44 @@ module.exports = function(worker, opts) {
   stopped.resolve(true);
 
   worker.send('xboard');
-
-  // Sunsetter always plays only moves right away. Count the number of played
-  // moves to show the correct mate in #n.
-  // When aiMoves is > 0 Sunsetter is in analysis mode, which uses an absolute
-  // eval score instead of relative to current color.
-  var aiMoves = 0;
-  var best, pv;
+  worker.send('reset crazyhouse');
 
   var processOutput = function(text) {
+    // console.warn("<-- %s", text);
```

Comment:
it's commented out, for what it's worth.

I left it to make it easy to quickly enable sunsetter output to console. Do you still want it removed?